VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionResponsesActivity.java

```diff
@@ -26,11 +25,4 @@ public Fragment getFirstFragment() {
 
         return courseDiscussionResponsesFragment;
     }
-
-    @Override
-    protected void onStart() {
-        super.onStart();
-
-        setTitle(R.string.course_discussion_responses_title);
```

Comment:
> All the logic is implemented in the fragment

The logic related to the Fragment is implemented there, this relates to the Activity :wink:

> and it'll be helpful in future as well, in case, the fragment gets added into another activity.

And that's just the thing; in a different Activity, we might not want to have it's title affected by what might be one of it's many Fragments. It's all about modularity.